src/state-summary/state-card-vacuum.js

```diff
@@ -0,0 +1,51 @@
+import '@polymer/iron-flex-layout/iron-flex-layout-classes.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+
+import '../components/entity/state-info.js';
+import '../components/ha-vacuum-state.js';
+import '../components/entity/ha-entity-toggle.js';
+
+class StateCardVacuum extends PolymerElement {
+  static get template() {
+    return html`
+    <style include="iron-flex iron-flex-alignment"></style>
+
+    <div class="horizontal justified layout">
+      ${this.stateInfoTemplate}
+      <template is="dom-if" if="[[supportsState(stateObj)]]">
```

Comment:
`[[_supportsState(stateObj.attributes.supported_features)]]`

wie prefix internal props and functions with `_`